rust/src/lib/nm/error.rs

```diff
@@ -37,6 +37,22 @@ pub(crate) fn nm_error_to_nmstate(nm_error: NmError) -> NmstateError {
                 ),
             )
         }
+        NmErrorKind::Connection(NmConnectionError::InvalidProperty) => {
+            if nm_error.msg.starts_with("ipv4.forwarding") {
+                NmstateError::new(
+                    ErrorKind::NotSupportedError,
+                    format!(
+                        "Please upgrade NetworkManager to 1.54 or above to \
+                        support IPv4 sysctl forwarding as indicated by: {nm_error}",
+                    ),
+                )
+            } else {
+                NmstateError::new(
+                    ErrorKind::Bug,
+                    format!("{}: {}", nm_error.kind, nm_error.msg),
+                )
+            }
```

Comment:
This is changing the error type and message that are used now, which are `            NmstateError::new(ErrorKind::InvalidArgument, nm_error.to_string())`. Either allow this to fallback to the existing lines below, or generate the same error.

Reply:
I changed it to fall back to the existing lines below. Honestly, I think that it is fine to keep it as `ErrorKind::Bug` when there are other `NmConnectionError::InvalidProperty` triggered but not related to `ipv4.forwarding`,  which means that there are some setting conflicts that are captured by NM but not properly validated in Nmstate yet.